parachain/runtimes/gargantua/src/ismp.rs

```diff
@@ -86,6 +95,58 @@ impl pallet_call_decompressor::Config for Runtime {
     type MaxCallSize = ConstU32<2>;
 }
 
+// todo: set corrrect parameters
+parameter_types! {
+    pub const AssetPalletId: PalletId = PalletId(*b"asset-tx");
+    pub const ProtocolAccount: PalletId = PalletId(*b"protocol");
+    pub const TokenGateWay: H160 = H160::zero();
+    pub const DotAssetId: H256 = H256::zero();
+    pub const ProtocolFees: Percent = Percent::from_percent(1);
```

Comment:
So actually i think these should be configurable at runtime and adjustable by root

Reply:
Yeah, was thinking of putting them in storage.